pages/pil/unscoped/courses/participants/read/views/training-pil.jsx

```diff
@@ -18,7 +18,7 @@ export default function TrainingPil({ className, trainingPil }) {
     <Inset className={className}>
       <ModelSummary
         model={{ ...trainingPil.trainingCourse, ...pick(trainingPil, 'expiryDate', 'issueDate') }}
-        schema={pick({ ...schema, ...trainingPilSchema }, fields)}
+        schema={{ ...pick({ ...schema, ...trainingPilSchema }, fields), projectId: { accessor: 'project.licenceNumber' } }}
```

Comment:
This is quite a lot of code for an inline prop declaration. Could you move it out into a `const schema = ...`?